### PR TITLE
Adv kit swerve sim

### PR DESCRIPTION
**Pull request by ChaoticChaosTheGreat**
We added AdvantageKit AutoLogging to the Swerve Subsystem (We need AdvantageScope but that can only be tested with SwerveSim)

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -33,6 +39,29 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+
+    Logger.recordMetadata("ProjectName", "MyProject"); // Set a metadata value
+
+    if (isReal()) {
+      Logger.addDataReceiver(new WPILOGWriter()); // Log to a USB stick ("/U/logs")
```
Comment (DragonXDev) [inline src/main/java/frc/robot/Robot.java:46]: Good usage of WPILOGWriter()

**Review by DragonXDev — APPROVED** [review]
Looks good! Some TalonFX motors don't seem completed because the swerve subsystem isn't completed but that can be fixed once it has.

**Review by DragonXDev — APPROVED** [review]
LGTM! So well written, I could've never written this myself.

**Review by realjoshuau — APPROVED** [review]
LGTM.

**Pull request merged**